rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -15,8 +15,8 @@ spec:
             sum by (source_cluster) (increase(kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod=~"tekton-.*|pipelines-as-code-.*"}[5m])) > 0
           for: 5m
           labels:
-            severity: critical
-            slo: "true"
+            severity: warning
+            slo: "false"
```

Comment:
`slo` is false by default so you may omit this.

Reply:
Also, you'll have to update the corresponding tests for this alert, they  are here: https://github.com/redhat-appstudio/o11y/blob/main/test/promql/tests/data_plane/pipeline_deadlock_crashloop_test.yaml

Reply:
Thanks! Updated.